### DISCUSSION TITLE
Installation Crashing at 14%

### DISCUSSION THREAD

**Discussion by ArchUser** [ventoy/Ventoy#2517] [Q&A]
I reviewed the log file, but I don't see what the issue is. Tested with Patriot 128GB and Patriot 256 GB Rage PRO drives. I also tried turning off Windows Defender, but that did not help. I also tried versions 1.94, 1.93, and 1.91 and all of them have the same issue.
[log.txt](https://github.com/ventoy/Ventoy/files/12297540/log.txt)

**Comment by dualkeyboards** [comment]
did you got any solution?
